assets/sass/config/_variables.scss

```diff
@@ -68,7 +68,7 @@ $c-gallery: #ebeef0;
 $c-gallery-2: #ebeef0;
 $c-fiord: #1b3870;
 $c-iron: #b8bdb9;
-$c-shark: #232824;
+$c-heavy-metal: #232824;
```

Comment:
This variable is renamed according to http://chir.ag/projects/name-that-color/. Regardless, this is unused in the codebase at the moment.

Reply:
@nfmohit I would revert the changes in this file and instead add the new color in 
`assets/sass/config/_variables-mui3.scss`
<img width="198" alt="image" src="https://user-images.githubusercontent.com/31818/212030352-77b8d2a0-b4c1-4e3b-b5d9-a6434119a5b5.png">

We can name it `$c-grey-g-900`?

Reply:
Updated, thank you @asvinb!